docs/en/standalone.md

```diff
@@ -0,0 +1,134 @@
+# Standalone Mode Guideline
+
+## Deployment
+### Modify Config
+1. Replace 127.0.0.1 with the real IP if may be accessed in another node
+2. If the specific port is unavailable, change to another port
```

Comment:
=>
2. (Optional) Please change the port number if it is occupied by other services.